packages/jaeger-ui-components/src/TracePageHeader/SpanGraph/Scrubber.tsx

```diff
@@ -19,50 +19,56 @@ import { createStyle } from '../../Theme';
 import { css } from 'emotion';
 
 export const getStyles = createStyle(() => {
-  const ScrubberHandleExpansion = css`
-    label: ScrubberHandleExpansion;
-    cursor: col-resize;
-    fill-opacity: 0;
-    fill: #44f;
-  `;
-  const ScrubberHandle = css`
-    label: ScrubberHandle;
-    cursor: col-resize;
-    fill: #555;
-  `;
-  const ScrubberLine = css`
-    label: ScrubberLine;
-    pointer-events: none;
-    stroke: #555;
-  `;
   return {
+    ScrubberHandleExpansion: cx(
+      css`
+        label: ScrubberHandleExpansion;
+        cursor: col-resize;
+        fill-opacity: 0;
+        fill: #44f;
+      `,
+      'scrubber-handle-expansion'
+    ),
+    ScrubberHandle: cx(
+      css`
+        label: ScrubberHandle;
+        cursor: col-resize;
+        fill: #555;
+      `,
+      'scrubber-handle'
+    ),
+    ScrubberLine: cx(
+      css`
+        label: ScrubberLine;
+        pointer-events: none;
+        stroke: #555;
+      `,
+      'scrubber-line'
+    ),
     ScrubberDragging: css`
       label: ScrubberDragging;
-      & .${ScrubberHandleExpansion} {
+      & .scrubber-handle-expansion {
         fill-opacity: 1;
       }
-      & .${ScrubberHandle} {
+      & .scrubber-handle {
         fill: #44f;
       }
-      & > .${ScrubberLine} {
+      & > .scrubber-line {
         stroke: #44f;
       }
     `,
     ScrubberHandles: css`
       label: ScrubberHandles;
-      &:hover > .${ScrubberHandleExpansion} {
+      &:hover > .scrubber-handle-expansion {
         fill-opacity: 1;
       }
-      &:hover > .${ScrubberHandle} {
+      &:hover > .scrubber-handle {
         fill: #44f;
       }
-      &:hover + .${ScrubberLine} {
+      &:hover + .scrubber.line {
         stroke: #44f;
       }
     `,
-    ScrubberHandleExpansion,
-    ScrubberHandle,
-    ScrubberLine,
   };
```

Comment:
These changes are not related to main change, just updated a PageHeader test and got some css emotion warnings that made reading the jest errors hard